codelabs/custom_generator/custom_generator.md

```diff
@@ -150,20 +169,11 @@ var codelabToolbox = {
 }
 ```
 
-Then update the options struct in the playground to use your new toolbox:
-
-```js
-var defaultOptions = {
-  comments: true,
-  // ...
-  toolbox: codelabToolbox,
-  // ...
-};
-```
+Our `index.js` file already handles importing the toolbox and using it in Blockly.
 
-Your toolbox should look like this:
+If the server is already running, you can refresh the page to see your changes. Otherwise, run `npm run start` to start the server. The app is still trying to generate and run JavaScript for the workspace, instead of JSON. We will change that soon. For now, you should see the new blocks in the toolbox, like this:
```

Comment:
Move "The app is still trying to generate and run JavaScript for the workspace, instead of JSON. We will change that soon." to after the screenshot, so that the flow is: 
1. Open it up. 
2. It should look like this. 
3. We still need to make more changes.